umbral/utils.py

```diff
@@ -55,6 +55,42 @@ def hash_to_bn(list, params):
  
     return res
 
+def hash_to_point(curve, data, constant=None):
```

Comment:
Let's change this name to `unsafe_hash_to_point`. As you point out, it isn't in constant time and we should be up front about the unsafe nature of this in the name as well as the code, too.

Reply:
:+1:

Reply:
Yep, I was thinking the same just now